humilis_elasticache/meta.yaml.j2

```diff
@@ -28,6 +28,18 @@ meta:
             value:
                 1
 
+        redis_cluster_mode:
+            description:
+                Activate or not cluster mode for redis.
+            value:
+                no
+
+        rep_per_group:
```

Comment:
I know I am not always consistent myself, but I'd rather avoid abbreviations unless they are very obvious. In this case I'd prefer `replicas_per_group`.